docs/learning/tutorials/develop-agent.md

```diff
@@ -2,43 +2,47 @@
 
 ### Setup
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
-- [Register an Agent Application](../registration-dashboard?tab=0&create=AGENT_APP)
+- [Register an Agent Application](../../../getting-started/registration-dashboard/?tab=0&create=AGENT_APP)
     - Accept all defaults
-- [Clone imodeljs-samples repo](https://github.com/imodeljs/imodeljs-samples)
+    - Be sure to save the secret somewhere safe - it is only shown once
+- [Clone agent-starter repo](https://github.com/imodeljs/agent-starter)
 - Have access to a cloud hosted iModel. If you do not have access to one, follow one of our tutorials to [create an online iModel]($docs/learning/tutorials/index.md)
-- Add {client_id}@apps.imsoidc.bentley.com as a project participant of your test iModel on the [iModel Registration Dashboard]($docs/getting-started/registration-dashboard)
- > Allow some time after registering the agent application. The identity profile of the agent is being created in the background and can take between 5 and 10 minutes.
+- Add {client_id}@apps.imsoidc.bentley.com as a project participant of your test iModel using the [iModel Registration Dashboard](../../../getting-started/registration-dashboard/?tab=1)
+ > Allow some time after registering the agent application. The identity profile of the agent is being created in the background and can take between 5 and 10 minutes
 
 
 ### Build
 - Open the cloned repo in VS Code
 - Open integrated terminal
-- `node ./common/scripts/install-run-rush install`
-- Edit agent-app/query-agent/src/QueryAgentConfig.ts
-    > `imjs_agent_imodel_name` = The name of your iModel<br/>
-`imjs_agent_client_id` = client_id of the registered agent application<br/>
-`imjs_agent_client_secret` = The client secret of registered agent application<br/>
-
+- `npm install`
+- Create a `.env` file at the project root with the following:
+``` ps

```

Comment:
Can you indent the code block 4 spaces and then remove the spaces from within the content?  That will make it a child of the bullet on line 18.